### ISSUE TITLE
What about a comparison with MiLe Loss?

### ISSUE THREAD

**Issue by suu990901** [PRIS-CV/EAFT#1]
Great work! Are you considering a comparison with MiLe Loss [https://arxiv.org/abs/2310.19531]? The loss formulation of MiLe Loss is similar to your approximation, and it was proposed for application in pre-training. Perhaps you should apply it and explain the differences.

**Comment by suu990901** [comment]
> Thanks for pointing this out! We apologize for missing MiLe Loss during our initial survey. As you noted, MiLe Loss primarily focuses on pre-training, while ours targets SFT. We find that the MiLe Loss formulation is mathematically close to the "Polynomial Scaling" variants analyzed in our ablation studies. We agree that including this comparison will significantly strengthen our work. We are launching experiments right now to compare them explicitly. We will cite MiLe Loss and include this discussion in our next revision. Best regards.

Looking forward to your experimental results. For the implementation of MiLe loss, you can refer to the link: https://github.com/suu990901/LLaMA-MiLe-Loss/blob/main/utils/trainer.py. If you need any other assistance, please let me know.

**Comment by ymxyll** [comment]
Thanks for the suggestion! We have added **MiLe Loss** as an additional ablation and report the results below.

Overall, we observe that MiLe tends to improve math-related performance, but is weaker at preserving general capabilities compared to EAFT. This is consistent with the design difference: MiLe mainly **amplifies learning on high-entropy (uncertain) tokens**, while EAFT explicitly **suppresses updates on low-entropy confident-conflict tokens**. In essence, MiLe is primarily oriented toward improving learning, whereas EAFT is geared toward mitigating forgetting.

| Method                | aime24 | aime25 | gsm8k | math_mean | mmlu  | ifeval | cluewsc | general_mean |
|-----------------------|:------:|:------:|:-----:|:---------:|:-----:|:------:|:-------:|:------------:|
| Qwen3-4B-Instruct-2507 | 63.30  | 47.40  | 94.30 |  68.33    | 77.08 | 80.96  |  85.20  |    81.08     |
| SFT                    | 63.30  | 50.00  | 94.80 |  69.37    | 76.51 | 79.48  |  74.45  |    76.51     |
| MiLE                  | 63.30  | 53.30  | 94.30 |  70.63    | 75.46 | 78.56  |  77.96  |    77.33     |
| EAFT                  | 60.00  | 53.30  | 94.50 |  69.27    | 76.59 | 80.10  |  83.66  |    80.12     |

We will include this MiLE ablation study in the next version of the manuscript. Thanks again!

**Comment by suu990901** [comment]
> Thanks for the suggestion! We have added **MiLe Loss** as an additional ablation and report the results below.
> 
> Overall, we observe that MiLe tends to improve math-related performance, but is weaker at preserving general capabilities compared to EAFT. This is consistent with the design difference: MiLe mainly **amplifies learning on high-entropy (uncertain) tokens**, while EAFT explicitly **suppresses updates on low-entropy confident-conflict tokens**. In essence, MiLe is primarily oriented toward improving learning, whereas EAFT is geared toward mitigating forgetting.
> 
> Method	aime24	aime25	gsm8k	math_mean	mmlu	ifeval	cluewsc	general_mean
> Qwen3-4B-Instruct-2507	63.30	47.40	94.30	68.33	77.08	80.96	85.20	81.08
> SFT	63.30	50.00	94.80	69.37	76.51	79.48	74.45	76.51
> MiLE	63.30	53.30	94.30	70.63	75.46	78.56	77.96	77.33
> EAFT	60.00	53.30	94.50	69.27	76.59	80.10	83.66	80.12
> We will include this MiLE ablation study in the next version of the manuscript. Thanks again!

Well done! We look forward to you **incorporating the MiLe results into Table 1 of your next version**. This will provide **greater insight** and enhance the understanding of the **differences** between EAFT and MiLe.
